src/components/Favourites/Favourites.component.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from 'react';
+import {
+  StyledCardsContainer,
+  StyledCardsGeneralContainer,
+  StyledWelcomeHeader,
+} from '../../styled/Cards.styled';
+import Card from '../Cards/Card.component';
```

Comment:
The path is not working correctly on my local machine can you take a look ? 
```suggestion
import Card from '../Cards';
```

Reply:
i think it was the s at the end, fixed it.